react/features/recording/components/LiveStream/StartLiveStreamDialog.web.js

```diff
@@ -341,6 +338,42 @@ class StartLiveStreamDialog extends Component {
             });
     }
 
+    /**
+     * Takes in a list of broadcasts from the YouTube API, removes dupes,
+     * removes broadcasts that cannot get a stream key, and parses the
+     * broacasts into flat objects.
+     *
+     * @param {Array} broadcasts - Broadcast descriptions as obtains from
+     * calling the YouTube API.
+     * @private
+     * @returns {Array} An array of objects that are flat and contain only data
```

Comment:
i don't think you need to explicitly mention that they are flat/contain the data that is needed here, do you?  you're returning an array of objects you're creating and define the format below.  i think it'd be better to say "return an array of objects describing a unique broadcast" (or whatever if that is inaccurate)